app/controllers/api/internal/purchases_controller.rb

```diff
@@ -13,6 +13,10 @@ def create
     respond_with Purchase.create!(purchase_params.merge(user: current_user))
   end
 
+  def update
+    respond_with purchase.update!(purchase_params)
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 17